src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -0,0 +1,95 @@
+<script>
+    /**
+     * A front-end representation of a Backpack field, with its main components.
+     *
+     * Makes it dead-simple for the developer to perform the most common
+     * javascript manipulations, and makes it easy to do custom stuff
+     * too, by exposing the main components (name, wrapper, input).
+     */
+    class CrudField {
+        constructor(fieldName) {
+            this.name = fieldName;
+            this.wrapper = $("[bp-field-name="+ this.name +"]");
+            this.input = $("[bp-field-main-input][name="+ this.name +"]");
```

Comment:
Notice this. If a field wants to define that its information is not ACTUALLY stored in the first input... but in a different input... it can do that, by just adding `bp-field-main-input` to that input/select/textarea.

I went through the main Backpack fields and it doesn't look like this is needed right now... all the fields that we have right now, if they store the info in a hidden field, that hidden field is the first one. But custom fields and future fields might not respect this convention - for those, we have `bp-field-main-input`.

Of course, I could also be wrong, and we _do_ have fields like that.